src/views/article.vue

```diff
@@ -45,6 +45,40 @@
                     </table>
                 </div>
             </div>
+            <!-- SUB PARAGRAPH TODO: create own class for sub paragraph-->
+            <div class="paragraph" v-for="(sub_paragraph) in chapter.sub_paragraphs" :key="sub_paragraph.index">
```

Comment:
please use camel case (also in the json)

Reply:
Thanks